apps/openmw/mwmechanics/mechanicsmanagerimp.cpp

```diff
@@ -1366,11 +1396,12 @@ namespace MWMechanics
 
         const MWMechanics::NpcStats& victimStats = victim.getClass().getNpcStats(victim);
 
-        // Simple check for who attacked first: if the player attacked first, a crimeId should be set
+        // Simple check for who attacked first:
+        // if the player or his follower/summon attacked first, a crimeId should be set.
         // Doesn't handle possible edge case where no one reported the assault, but in such a case,
         // for bystanders it is not possible to tell who attacked first, anyway.
         if (victimStats.getCrimeId() != -1)
-            commitCrime(attacker, victim, MWBase::MechanicsManager::OT_Murder);
+            commitCrime(attackerIsPlayerFollowerOrSummon ? player : attacker, victim, MWBase::MechanicsManager::OT_Murder);
```

Comment:
I've tested `startcombat, npc_id` on the summoned skeleton and high pockets reference and after it killed the NPC it got me 1000 bounty with vanilla. Also I've checked if after I attacked an NPC, and then summon or high pockets killed it, i also got 1000 bounty. The same after I've summoned summon after attacking the NPC, it got me 1000 bounty.